bedrock/products/models.py

```diff
@@ -109,3 +115,197 @@ def get_context(self, request, *args, **kwargs):
+    template = "products/monitor/cms/index.html"
+
+    split_heading = models.CharField(max_length=255, blank=False)
+    split_subheading = models.CharField(max_length=255, blank=True)
+    split_content = RichTextField(null=True, blank=True, features=["bold", "italic", "link", "ul", "ol"])
+    split_button_text = models.CharField(max_length=255, blank=False)
+    split_button_link = models.URLField(null=True, blank=False)
+    split_image = models.ForeignKey(
+        "cms.BedrockImage",
+        null=True,
+        blank=False,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    articles_heading = models.CharField(max_length=255, blank=False)
+    articles_subheading = models.CharField(max_length=255, blank=True)
+
+    call_to_action_bottom = models.ForeignKey(
+        "products.MonitorCallToActionSnippet",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    content_panels = AbstractBedrockCMSPage.content_panels + [
+        MultiFieldPanel(
+            [
+                FieldPanel(
+                    "split_heading",
+                    heading="Intro Heading",
+                ),
+                FieldPanel(
+                    "split_subheading",
+                    heading="Intro Subheading (optional)",
+                ),
+                FieldPanel(
+                    "split_content",
+                    heading="Intro Content (optional)",
+                ),
+                FieldRowPanel(
+                    [
+                        FieldPanel("split_button_text", heading="Intro Button text"),
+                        FieldPanel("split_button_link", heading="Intro Button link"),
+                    ]
+                ),
+                FieldPanel("split_image", heading="Intro Image"),
+            ],
+            heading="Intro Split Block",
+        ),
+        MultiFieldPanel(
+            [
+                FieldPanel("articles_heading"),
+                FieldPanel("articles_subheading"),
+            ],
+            heading="Article Section Heading",
+        ),
+        HelpPanel(content="The articles will be linked to here"),
+        FieldPanel("call_to_action_bottom"),
+    ]
+
+    def get_context(self, request):
+        context = super().get_context(request)
+        context["articlepages"] = MonitorArticleDetailPage.objects.live().public()  # live() and public() are wagtail filters
+        return context
+
+
+class MonitorArticleDetailPage(AbstractBedrockCMSPage):
+    AbstractBedrockCMSPage._meta.get_field("title").verbose_name = "Heading"
+    AbstractBedrockCMSPage._meta.get_field("title").help_text = "Heading (use sentence case)"
+
+    template = "products/monitor/cms/article.html"
+    parent_page_types = ["MonitorArticleIndexPage"]  # must be child of MonitorArticleIndexPage
+
+    # promote panels
+    icon = models.ForeignKey(
```

Comment:
For SVG icons, one easier option we have (rather than having to step around the rendtion machinery is to just use a regular `ImageField` or `FileField` with a custom valiator on it to ensure SVG only

Something like this might work (from our gpt robot colleague)

```
def validate_image_extension(value):
    allowed_extensions = [".svg",]
    if not any(value.name.lower().endswith(ext) for ext in allowed_extensions):
        raise ValidationError("Only SVG images are allowed.")

....
    file = models.ImageField(upload_to="images/svgs", validators=[validate_image_extension])

````

Or we stick with image fields and handle it differently in the template

Reply:
We would not need a rendition of the SVG and can directly reference it, so that is my preference. I'd rather give the editors a list of valid options rather than throwing an error if they pick a bad one out of a list.

Reply:
We can add help_text on the field to give editors a nudge to ensure they upload an SVG into that FileField, if that's what you're contemplating here